openfisca_web_api/loader/parameters.py

```diff
@@ -95,17 +95,21 @@ def build_parameters(tax_benefit_system, country_package_metadata):
     api_parameters = []
     walk_node(
         original_parameters,
-        parameters = api_parameters,
-        path_fragments = [],
-        country_package_metadata = country_package_metadata,
+        parameters=api_parameters,
+        path_fragments=[],
+        country_package_metadata=country_package_metadata,
         )
 
     return {parameter['id'].replace('.', '/'): parameter for parameter in api_parameters}
 
 
 def build_parameters_overview(parameters):
+
     return {
-        parameter['id']: {'description': parameter['description']}
+        parameter['id']: {
+            'description': parameter['description'],
+            'href': 'parameter/' + parameter['id']
```

Comment:
Prefer using `format` as `+` can quickly become hard to read (... until we can use the nice Python 3.6 new syntax!)